Jenkinsfile.baguette

```diff
@@ -87,12 +87,22 @@ pipeline {
                     steps {
                         dir('src/github.com/docker/app') {
                             checkout scm
+                            dir('_build') {
```

Comment:
Why is there jenkins related stuff in this PR?
Seems like something coming from another branch.
(and seems like it is breaking the merge-pr build)

Reply:
The changes are needed in order for Coverage to run normally, the thing kinda worked because `Test Linux` and `Coverage` ran on the same slave and `Coverage` will take the invocation-image loaded from the `Test Linux` step. 

It's not related to the refatoring, I can make a new PR with only this change if you want.